### PR TITLE
Removing Country Code DropDown from Phone Question

### LINKED ISSUE #6525: Default country code for phone numbers
#### Is your feature request related to a problem? Please describe.

Most of our applicants (if not all of them!) will have a U.S. phone number. Is it possible to set the "Country" on the phone number question type to default to "United States" automatically? Alternatively, could we just hide the "Country" selector? It's an extra question that doesn't impact the input of the phone number; that is, no matter what the person selects, they are still prompted to enter a 10-digit number. 

#### Describe the solution you'd like.

As an applicant, I do not want to answer the "County" question when asked for my phone number.

#### What priority is this?

P2-we'd like to switch out the phone number question that is currently in use in all program forms (currently just a text question type) with the actual phone number question type. But it would be helpful if that change doesn't add an additional question for residents to complete. So we'd like to remove the "Country" question before switching the questions in all program forms. 

We would also like to align the format of the Seattle Login phone number question with the CiviForm phone number question--both should be 10-digit phone numbers. The Seattle Login does not ask for country code, so this information is extraneous.

#### Associated civic entity

Seattle

#### User Group

Feedback is coming from the CiviForm Admins. Residents have not yet tested this feature.

#### Additional Context
Current State and Options
image

Seattle Login Phone Formatting:
image

#### Done when:

Applicants no longer need to select a Country.

### PR DESCRIPTION
**Pull request by gwendolyngoetz**
### Description

Removing Country Code DropDown from Phone Question. Additionally this fixes the input formatting when there are multiple phone number questions on the same screen.

We now call the new method which will determine the County Code and it's validity based on the supported Country Code options (US and CA at this time).

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers).
- [x] Updated units where needed
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)

### Issue(s) this completes

Fixes #6525

### CODE REVIEW

File: server/app/assets/javascripts/phone.ts [hunk 1]
```diff
@@ -19,11 +19,13 @@ function formatPhone(element: HTMLElement) {
 }
 
 export function init() {
-  const element = document.getElementById('cf-phone-number')
-  const countryElement = document.getElementById('cf-phone-country-code')
-  if (element && countryElement) {
-    formatPhone(element)
-  }
+  document.querySelectorAll('[id^=cf-phone-number]').forEach((el) => {
+    // Our questions automatically include an element matching id appended with "-errors"; filter them out.
```
Comment (rockycodes) [inline server/app/assets/javascripts/phone.ts:23]: praise: helpful comment! And good to know.

Comment (rockycodes) [reply server/app/assets/javascripts/phone.ts:23]: suggestion (non-blocking): add a sentence to this comment explaining that the phone elements will have a unique id with a number prepended on the end which is why you have to select them this way and then filter out the `error` ones.

File: server/app/views/questiontypes/PhoneQuestionRenderer.java [hunk 2]
```diff
@@ -78,26 +56,16 @@ protected DivTag renderInputTags(
                     phoneQuestion.getPhoneNumberPath(), ImmutableSet.of()))
             .setScreenReaderText(applicantQuestion.getQuestionTextForScreenReader())
             .addReferenceClass(ReferenceClasses.PHONE_NUMBER)
```
Comment (rockycodes) [inline server/app/views/questiontypes/PhoneQuestionRenderer.java:58]: question (non-blocking): do we still need this reference class?

**Review by rockycodes — APPROVED** [review]
looks great! just a couple of non-blocking comments.

File: server/app/views/questiontypes/PhoneQuestionRenderer.java (hunk 2 above)
Comment (gwendolyngoetz) [reply server/app/views/questiontypes/PhoneQuestionRenderer.java:58]: At least for now. A lot of tests started to fall apart when I removed/changed it.

**Pull request merged**